source/mongodb/internal-database-usage.txt

```diff
@@ -4,6 +4,14 @@
 Internal Database Usage
 =======================
 
+.. meta::
+   :description: Learn about how Device Sync interacts with your linked Atlas clusters.
+   :keywords: Realm
```

Comment:
I'm not sure this keyword is right either - nothing on this page seems to be directly related to Realm, and only one of them is related to Device Sync, which is also - not analogous to Realm?